__tests__/userJourney/userJourneyInit.spec.ts

```diff
@@ -0,0 +1,12 @@
+import Perfume from '../../src/perfume'
+import { config } from '../../src/config';
+
+import { testConfig } from '../../src/userJourneyTestConstants'
+
+describe('initPerformanceVitals', () => {
```

Comment:
This `initPerformanceVitals` has not reference with Perfume.

All the config tests are in https://github.com/Zizzamia/perfume.js/blob/master/__tests__/perfume.spec.ts#L34

Plus we need one unit test for every configuration.

The reason are called Unit, is because we define tests by each Unit. So that if something is wrong we just fix the Unit that is not working.

Have tests that are in bulk are easier to write, but make the library harder to mantain.